thoth/storages/graph/enums.py

```diff
@@ -70,3 +70,13 @@ class QuerySortTypeEnum(Enum):
 
     PACKAGE_NAME = "package_name"
     PACKAGE_VERSION = "package_version"
+
+
+class ThothAdviserIntegrationEnum(Enum):
+    """Class for source type enum."""
+
+    CLI = "CLI"
```

Comment:
```
from enum import auto
```
you can use `auto()`.
example `CLI = auto()`

Reply:
`ENUM` class doesn't support auto(), that is if you pass a enum class with as above, it seems to give an error like this - https://github.com/thoth-station/storages/pull/1859#discussion_r447112712

Reply:
Based on the error:

```
TypeError: object of type 'int' has no len()
```

you've probably passed value of the enum to the `source_type` enum on postgres side:

```python
    source_type = Column(
        ENUM(
            ThothAdviserIntegrationEnum.CLI.value,
            ThothAdviserIntegrationEnum.KEBECHET.value,
            ThothAdviserIntegrationEnum.S2I.value,
            ThothAdviserIntegrationEnum.GITHUB_APP.value,
            ThothAdviserIntegrationEnum.JUPYTER_NOTEBOOK.value,
            name="source_type",
            create_type=True,
        ),
        nullable=True,
    )
```

That will turn name into an `int` based on `auto()` invocation. You can provide enum names (using this trick should not enforce their enumeration every time):

```python
    source_type = Column(
        ENUM(
            *(e.name for e in ThothAdviserIntegrationEnum),
            name="source_type",
            create_type=True,
        ),
        nullable=True,
    )
```

Reply:
Thanks! auto() does seem to work now after passing `.name`. 
> (using this trick should not enforce their enumeration every time):
Sorry I didn't understand how this would affect the enumeration?

Reply:
> Sorry I didn't understand how this would affect the enumeration?

No need to list all the enums in the `ENUM()`. If we add another one, it will be automatically populated there.